### DISCUSSION TITLE
Preview commit URL works, but Branch URL does not resolve

### DISCUSSION THREAD

**Discussion by benripka** [vercel/community#4553] [Help]
### Summary

Hello, I have a vercel site up and a branch called test. I pushed to test the other day which automatically generated a preview site at https://<branch_url>.vercel.app/

It worked fine for a day or two but then the DNS no longer seems to resolve for it. I tried redeploying via git and Vercel, and have found that the new commit subdomain url https://<commit_subdomain>.vercel.app/  works fine but the test branch subdomain still doesn't lead anywhere! 

Does anyone have any idea what might be happening? I like to use the test branch for preview because it stops me from needing to update the redirect URI lists for my OAuth providers everytime...

Let me know if you need any more info or logs to help troubleshoot :) Thanks for your time

### Steps to Reproduce

Create a branch called test. Push to it and have vercel create a preview deployment. Leave that preview deployment up for 3 days and see if the preview branch subdomain URL still resolves. Redeploy and verify that the commit subdomain URL still resolves while the branch subdomain URL does not.

**Comment by amyegan** [comment]
Can you walk me through the steps you took to configure the branch domain?  If you also share the URL, I can look into it more.
